### ISSUE TITLE
Restore Settings fails 

### ISSUE THREAD

**Issue by ga3a** [EasyFarm/EasyFarm#360]
When restoring from file, nothing shows up in the main program.

It seems to be loaded in, but the different menus under "Battle" are not filled in

**Comment by Sinnersslayer** [comment]
You have to switch the tab to another and go back to Battle.
It will be filled .

**Comment by Mykezero** [comment]
Yes, this is a bug in the application when you load a profile that the interface doesn't update until you switch tabs.
